src/DataFixtures/Lector.php

```diff
@@ -14,7 +14,7 @@ class Lector extends Fixture implements OrderedFixtureInterface
     public function load(ObjectManager $manager)
     {
         /** @var UserEntity $user */
-        $user = $this->getReference(User::USER_REF);
+        $user = $this->getReference(User::USER_REF4);
```

Comment:
tady bys pak vytahoval         
`$user = $this->getReference(sprintf(%s_%s, User::USER_REF, 4)); `